genie_registry/maf.py

```diff
@@ -277,6 +277,25 @@ def _validate(self, mutationDF):
 
         return total_error.getvalue(), warning.getvalue()
 
+
+    def _cross_validate(self, mutationDF: pd.DataFrame) -> tuple:
```

Comment:
First step:
1. You have a list of ancillary files
2. Loop through them to find "data_clinical_supp_{center}.txt" OR "data_clinical_supp_sample_{center}.txt"
3. Once you find this ^, `from clinical import clinical, clinical()._read_data(...)`, <- if this doesn't work, hard code `try catch pd.read_csv(sep="\t")`
5. pass the maf df and sample df into the comparison function